modules/cnv_master/1.0/cnv_master.smk

```diff
@@ -169,6 +168,8 @@ rule _cnv_master_output_capture_merges:
 
 rule _cnv_master_all:
     input:
+        expand(_get_all_segs("genome"), projection = ["grch37", "hg38"]),
```

Comment:
Can we update this projection specification to be `CFG["projections"]`, so it is dynamic and pulls down the projections from config?

Reply:
Great idea - done!